material/admin/templates/admin/includes/material_js.html

```diff
@@ -3,6 +3,7 @@
 {% if not debug %}
 <script src="{% static 'material/js/materialize.admin.min.js' %}"></script>
 {% else %}
+<script src="{% static 'material/js/materialize.forms.js' %}"></script>
```

Comment:
In this file `matererialize.forms.js` are included on the line 14 already.

And it's too premature to include it at the first line before other jQuery plugins.